frontend/packages/client/src/components/ProposalCreate/StepOne/index.js

```diff
@@ -333,10 +409,11 @@ const StepOne = ({
             editorState={localEditorState}
             toolbarClassName="toolbarClassName"
             wrapperClassName="border-light rounded-sm word-break-all"
-            editorClassName="px-4"
+            editorClassName="px-4 content"
```

Comment:
this is a new class added because of this: https://bulma.io/documentation/elements/content/
elements like `h1` were being overwritten by not having the content class as a wrapper.